### PR TITLE
RFC: Introduce UI state cookie

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by obulat — APPROVED** [review]
Thank you for the write-up!

File: rfcs/20220218-introduce_ui_state_cookie_to_frontend.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# RFC: Introduce UI state cookie to frontend
+
+- [ ] Approver (TBD)
```
Comment (sarayourfriend) [inline rfcs/20220218-introduce_ui_state_cookie_to_frontend.md:3]:
```suggestion
- [ ] @obulat 
```

File: rfcs/20220218-introduce_ui_state_cookie_to_frontend.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# RFC: Introduce UI state cookie to frontend
+
+- [x] @obulat 
+- [ ] Approver (TBD)
```
Comment (sarayourfriend) [inline rfcs/20220218-introduce_ui_state_cookie_to_frontend.md:4]: @zackkrida based on https://github.com/WordPress/openverse-frontend/issues/909#issuecomment-1044871690 can I mark you as an approver?

```suggestion
- [x] @zackkrida 
```

**Review by zackkrida — APPROVED** [review]
lgtm!

The implementation plan looks straightforward, I think we can go ahead and create new issues.

File: rfcs/20220218-introduce_ui_state_cookie_to_frontend.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# RFC: Introduce UI state cookie to frontend
+
+- [x] @obulat 
+- [x] @zackkrida 
+
+**Milestone:** TBD
```
Comment (sarayourfriend) [inline rfcs/20220218-introduce_ui_state_cookie_to_frontend.md:6]:
```suggestion
**Milestone:** https://github.com/WordPress/openverse-frontend/milestone/11
```

**Pull request merged**